helpers/milestones.js

```diff
@@ -24,5 +24,5 @@
  * @todo Add description for the namespace and the properties
  */
 module.exports = {
-	disableDappTransfers: 0,
+	disableDappTransfers: 5594491,
```

Comment:
Isn't this the last block in a round?

Reply:
It is. It will be the first block forged on by 1.0.0 version.